profile/README.md

```diff
@@ -1,7 +1,7 @@
 # Backbase Open Source
 
-Welcome to Backbase's open source organisation. Regardless to if you are a customer, partner or employee you can create and share with everyone in the Backbase Community via this organisation.
+Welcome to Backbase's open source organisation. Regardless of if you are a customer, partner, or employee you can create and share with everyone in the Backbase Community via this organisation.
```

Comment:
Maybe `Regardless of if ...` -> `Regardless if` or `Regardless of whether`? (Because is sounds little weird otherwise.)

Reply:
The heading is `Backbase Open Source`, not `Backbase's Open Source`; hence, would be `Backbase Open Source organisation` instead.

Reply:
I agree, done ✅

Reply:
Would you mind while updating to also change the links?

https://backbase.io/developers/

and

https://backbase.io/learning/

Also, I believe there are 2 versions of this text, one for the public view and one for the members. So this should be copied their too.

Thank you

Reply:
@craigwalkeruk links are updated ✅

[I searched for the title of the README on GitHub to find a second page/text](https://github.com/search?q=Welcome+to+Backbase%27s+open+source+organisation.+Regardless+to+if+you+are+a+customer%2C&type=code) but this is the only file I found so far. 
Maybe you could share a link to the 2nd file that should be updated as well?

Reply:
Hi @ugur-bb @craigwalkeruk, 
I adressed all comments, except one about the second version of this text that needs to be updated. Unfortunately I couldn't find it myself. Could you please help with it?